builtin/providers/cidr/datasource_cidr_network.go

```diff
@@ -0,0 +1,117 @@
+package cidr
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type subnetMask struct {
+	Name string
+	Mask int
+}
+
+func dataSourceNetwork() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNetworkRead,
+
+		Schema: map[string]*schema.Schema{
+			"cidr_block": &schema.Schema{
+				Type:        schema.TypeString,
+				Description: "The CIDR Block for the entire network (aka supernet)",
```

Comment:
block (in lowercase)

Reply:
fixed in https://github.com/moofish32/terraform-provider-cidr/pull/2